admin/section-column-resize.php

```diff
@@ -40,9 +43,11 @@
 
 		$block_sizes[] = (int) $block_columns;
 
-		$block_increment++;
+		$block_increment ++;
 	}
-endif;
-if ( (int) $section_blocks > 1 && (int) $section_blocks < 4 ) : ?>
+}
+
+if ( $section_blocks > 1 && $section_blocks < 4 ) : ?>
```

Comment:
Honestly this just comes down to "type casting". A more efficient way would probably be to intval( $section_blocks ) instead of using (int) twice.